beacon_node/eth2_libp2p/Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Sigma Prime <contact@sigmaprime.io>"]
 edition = "2018"
 
 [dependencies]
-discv5 = { version = "0.1.0-beta.3", features = ["libp2p"] }
+discv5 = { git = "https://github.com/sigp/discv5 ", branch = "enr-loop", features = ["libp2p"] }
```

Comment:
We should use the hash in master, so that when the branch gets removed this commit still works

Reply:
Waiting on https://github.com/sigp/discv5/pull/63 to merge then I can use the master commit.